ssn/chapters/System-capabilities.html

```diff
@@ -909,6 +912,40 @@ <h5><a href="#SSNSYSTEMResolution">ssn-system:Resolution</a></h5>
         </div>
     </section>
 
+    <section class="specterm sosa" id="SSNSYSTEMRange" about="ssn-system:Range" typeof="owl:Class">
+        <h5><a href="#SSNSYSTEMRange">ssn-system:Range</a></h5>
+        <p class="crossreference"><strong>IRI:</strong> http://www.w3.org/ns/ssn/systems/Range</p>
+        <p><strong>an OWL Class</strong></p>
+
+        <em property="rdfs:label">Range</em> -
+        <span property="rdfs:comment skos:definition">A primitive class to represent that the value is within a numerical range.</span>
+        <p>The use of <code>schema:minValue</code> and <code>schema:maxValue</code> are suggested to record the bounds of the numerical range.</p>
```

Comment:
I rather have the canonical OWL/XSD facets for this. See section 3.3.6: 
https://linked-sdi.com/LinkedSDI.html#literals-facets

Reply:
I see a problem here in that their implementation lies in using owl:Restrictions. This makes sense in the case of an output value that can never exceed the range, but in this case the actual environmental reading can exceed the range.

Reply:
Rob has clarified the requirement here, which is that an instance may need to state a value outside of the range specified in an OWL constraint. So OWL facets don't work here.

Reply:
@ldesousa Can I have your 0.02$ on this?

Reply:
Two questions here:
- `xsd:minInclusive` and `schema:minValue` are semantically equal, therefore I would always opt for the W3C standard.
- `owl:withRestrictions` is more of a design option. I would still prefer to have it, what is the semantic function of a limit that can be surpassed? Perhaps the property itself needs a some more thinking? Two kinds of limits?

Reply:
@ldesousa The issue lies in the sensor class vs sensor instance problem. 

In _a sensor class_, ```owl:withRestrictions``` make sense in that the sensor is not designed to handle environments outside these values. It's a specification and an ideal model, owl forces the enforcement.

In a _Sensor with Serial Number of a sensor class_, bad things happen to sensor out in the field and out of bounds values happen all the time, be it as magic numbers or through fault.  If we can't list values that are reported by the sensors and out of bounds, it inhibits the capacity of users to ensure data quality.

You are right that xsd:minInclusive and schema:minValue are semantically equal and I have no concerns about using xsd:minInclusive, but feel we should enforce this throughout the specs.

I feel the solution is that the range should list  xsd:minInclusive / xsd:maxInclusive as properties and leave the ontological property enforcement to the user according to the application.  I'll add this to the documentation.

Reply:
Is not `sh:minInclusive` / `sh:minExclusive` (from [SHACL](https://www.w3.org/TR/shacl/#core-components-range) more useful than `xsd:minInclusive`? The SHACL form is also a W3C standard also but one for which there is more modern Semantic Web tooling available.

Reply:
We have not utilised SHACL in this edition. 
The 2017 edition was OWL/RDFS, so we decided to stick with that for this content-focussed update. 

However, I would strongly endorse the development of a SHACL implementation of the SSN Ontology, as a separate standalone document, 'Part 2'. 

Note that https://github.com/w3c/sdw-sosa-ssn/issues/93 mentions some SHACL implementation work undertaken by OGC.

Reply:
> ... and leave the ontological property enforcement to the user according to the application. I'll add this to the documentation.

That statement is crucial. +1 on this

> SHACL

+1 as well 

> as a separate standalone document, 'Part 2'

@dr-shorthair  : in order to deliver soon would your statement imply that 
- A°/ there should be no xsd:minInclusive awaiting for that Part2 ?
- or B°/ that 1st we should deliver with xsd:minInclusive then update with sh:minInclusive when that Part2 is available ?

Reply:
B. 
Use OWL/RDFS/XSD in this document. 
Provide alternative, but 'equivalent' SHACL implementation in a separate document. 

This should be easier than the work we are doing now as it will be largely mechanical.

Reply:
And the SHACL version is likely to be the most used implementation very quickly ...